packages/frontend/src/components/TableTree.styles.ts

```diff
@@ -10,6 +10,21 @@ export const TableTreeGlobalStyle = createGlobalStyle`
           white-space: normal;
         }
     }
+    .bp4-tree-node-content-0 {
+        padding-left: 0;
+    }
+    .bp4-tree-node-content-1 {
+        padding-left: 20px;
```

Comment:
Had to start from 20 the first level, otherwise the arrow would be cut off when there was only 1 table.